specification/loadtestservice/data-plane/loadtesting/models.tsp

```diff
@@ -1386,6 +1479,44 @@ union RequestDataLevel {
   ERRORS: "ERRORS",
 }
 
+// TODO(mitsha): Fix the structure of initial clone test return
+@added(APIVersions.v2025_03_01_preview)
+@doc("Operation Id Path Parameter.")
+model OperationIdPathParameter {
```

Comment:
nit, I don't know why this is separated from `model OperationStatus`. For a `@resource`, the `@key` property should be defined with the resource.

(and you set it to `visibility=Read`, hence this is not only a `PathParameter` -- it would be in response body)

Reply:
Done